.github/workflows/publish-helm-charts.yml

```diff
@@ -0,0 +1,98 @@
+name: Publish Helm OSS charts
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+
+on:
+  push:
+    branches: 
+      - 'master'
+    paths:
+      - 'charts/**'
+  workflow_dispatch:
+
+jobs:
+  release-chart:
+    name: Chart release
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          path: "airbyte"
+          fetch-depth: 0
+
+      - uses: cperezabo/setup-git-mkver@v1
+
+      - uses: actions/checkout@v3
+        with:
+          repository: "airbytehq/helm-charts"
+          token: ${{ secrets.OCTAVIA_PAT }}
+          path: "airbyte-oss"
+
+      - name: Generate release number
+        shell: bash
+        id: next-version
+        working-directory: ./airbyte
+        run: |
+          git fetch --depth=1 origin +refs/tags/*:refs/tags/*
+          echo "::set-output name=number::$(git mkver -c ./mkver.conf next | cut -d '+' -f1)"
+
+      - name: Mkver tag.
+        shell: bash
+        working-directory: ./airbyte/charts
+        run: |
+          git config user.name "github-actions[bot]"
+          git config user.email "41898282+github-actions[bot]@users.noreply.github.com"
+          git mkver -c ../mkver.conf next
+          git mkver -c ../mkver.conf tag
+          git mkver -c ../mkver.conf patch
+
+
+      - name: Create tag
+        uses: actions/github-script@v5
+        with:
+          script: |
+            github.rest.git.createRef({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              ref: 'refs/tags/v${{ steps.next-version.outputs.number }}+helm',
+              sha: context.sha
+            })
+
+      - name: "Helm package"
+        shell: bash
+        run: |
+          declare -a StringArray=("airbyte-bootloader" "airbyte-server" "airbyte-temporal" "airbyte-webapp" "airbyte-pod-sweeper" "airbyte-worker")
+          for val in ${StringArray[@]}; do
+            cd ./airbyte/charts/${val} && helm dep update && cd $GITHUB_WORKSPACE
+            helm package ./airbyte/charts/${val} -d airbyte-oss  --version ${{ steps.next-version.outputs.number }}
+          done
+          helm repo index airbyte-oss/
+      
+      - name: Commit and push changes
+        uses: EndBug/add-and-commit@v9
+        with:
+          message: 'Bump release to ${{ steps.next-version.outputs.number }}+helm'
+          add: '.'
+          cwd: './airbyte-oss/'
+
+      - name: "Helm package main chart"
+        shell: bash
+        run: |
+          sleep 120
+          declare -a StringArray=("airbyte")
+          for val in ${StringArray[@]}; do
+            cd ./airbyte/charts/${val} && cat Chart.yaml && helm dep update && cd $GITHUB_WORKSPACE
```

Comment:
to make sure i understand, `helm dep update` means we always update our chart dependencies before publishing?

Reply:
We need to update and download dependencies of chart before publishing it(it packages everything into single tarball archive)